core/templates/dev/head/components/AnswerGroupEditorDirective.js

```diff
@@ -208,9 +208,13 @@ oppia.directive('answerGroupEditor', [function() {
         };
 
         $scope.cancelActiveRuleEdit = function() {
-          $scope.rules.splice(0, $scope.rules.length);
-          for (var i = 0; i < $scope.rulesMemento.length; i++) {
-            $scope.rules.push($scope.rulesMemento[i]);
+          if ($scope.rulesMemento !== null) {
```

Comment:
I reckon I must have screwed something up initially. Looking at the commits, it looks like they begin in my previous PR. Any way to undo this?